inductiva/molecules/utils.py

```diff
@@ -42,3 +44,19 @@ def align_trajectory_to_average(universe, trajectory_output_path):
                     select="protein and name CA",
                     filename=trajectory_output_path,
                     in_memory=False).run()
+
+
+def download_pdb_from_rcsb(pdb_id: str, save_dir: Optional[str] = None) -> str:
+    """Download a PDB file from the RCSB database according 
+    to its pdb ID.
+    Args:
+        pdb_id: The PDB identifier of the molecule to download.
```

Comment:
What happens if I pass a non-existing pdb_id?

Reply:
You will receive a message saying this file does not exist.

Reply:
Yes, seems like it will get to:

`logging.error("Could not download file from %s", url)`

You know what happens next?